app/models/chargeback.rb

```diff
@@ -170,4 +170,8 @@ def self.get_report_time_range(options, interval, tz)
   def self.report_cb_model(model)
     model.gsub(/^Chargeback/, "")
   end
+
+  def self.chargeback_model?(model)
+    %w(ChargebackContainerProject ChargebackVm).include?(model)
```

Comment:
@simon3z : I wanted to enumerate the subclasses rather then match them with `start_with?` so that when we refactor this for provider plugability we will have an idea on what might actually come here